### DISCUSSION TITLE
Qexo2.0发版计划

### DISCUSSION THREAD

**Discussion by am-abudu** [Qexo/Qexo#157] [Announcements]
主题: 拓展性
1. 将function内对于github的专门适配代码一般化，并将github gitlab等适配部分放入provider文件夹供function调用来实现provider的拓展性
2. 同1实现图床的拓展性
3. 支持用户从并非github的其他渠道一键更新（但不能超时）
4. 尝试将js css整体换到cdnjs以应对未来的不定情况
5. 完善首次部署逻辑

若没有意外情况，将在7~8月份进行重构
若有其他需要的功能，建议在此处留言

**Comment by Fgaoxing** [comment]
@am-abudu good

**Reply by L-20021213** [reply]
希望能加一个打断github actions的功能。

**Reply by Fgaoxing** [reply]
需求有点少啊，你问abudu，估计是不行

**Comment by Fgaoxing** [comment]
个人建议
1. [国际化](https://github.com/Qexo/Qexo/issues/107#issuecomment-1065823984)
2.  #155 
3. [Front-matter](https://hexo.io/zh-cn/docs/front-matter.html)可视化操作，像hexon那样
4. 说说

**Reply by Fgaoxing** [reply]
@am-abudu

**Reply by Fgaoxing** [reply]
一些建议的整合
#140 
#147 
#158

**Comment by L-20021213** [comment]
我直接Fork仓库，全局替换反代，不准备更新了，能用就行了。

**Reply by Fgaoxing** [reply]
默认cdn早就换了：[e275903](https://github.com/Qexo/Qexo/commit/e27590327c15d44c7766d0e38f12fd5912301c83)

**Reply by am-abudu** [reply]
编辑高级设置中的'CDN_PREV'可一键替换

**Comment by Liynw** [comment]
~~我就说为啥我的后台突然 500 了~~

个人希望添加的功能：
1. 增加 GitHub / NPM 图床的功能，可以管理指定 GitHub 仓库文(tu)件(pian) **和更新 `package.json`**；
2. 优化一下自适应，我用的小笔记本访问后台貌似不能有很好的观感，调到 90% 也不行 qwq。

谢谢 abudu 大佬！

**Reply by am-abudu** [reply]
github图床之前因为github的相关政策说过不会支持的，npm同理，但届时你可以自己写provider就是了。。

**Reply by Fgaoxing** [reply]
#71 #31

**Reply by imjason** [reply]
package.json可以加个

**Reply by am-abudu** [reply]
关于第二点 可以尝试点编辑器上面那个小画笔，可以修改模式

**Comment by Fgaoxing** [comment]
多主题支持(望采纳)：https://www.creative-tim.com/product/soft-ui-dashboard-django
https://www.creative-tim.com/product/material-dashboard-django
https://www.creative-tim.com/product/black-dashboard-django

**Reply by Fgaoxing** [reply]
还有开放友链适配加入文档
